docs/architecture/adr-056-proving-amnesia-attack.md

```diff
@@ -0,0 +1,106 @@
+# ADR 056: Proving flip flopping attacks
+
+## Changelog
+
+- 02.04.20: Initial Draft
+- 06.04.20: Second Draft
+
+## Context
+
+Whilst most created evidence of malicious behaviour is self evident such that any individual can verify them independently there are types of evidence, known as global evidence, that require further collaboration from the network in order to accumulate enough information to create evidence that is individually verifiable and can therefore be processed through consensus. Fork Accountability as a whole constitutes of detection, proving and punishing. This ADR addresses how to prove infringement from global evidence that is sent to a full node.
+
+The currently only known form of global evidence stems from [flip flopping](https://github.com/tendermint/spec/blob/master/spec/consensus/light-client/accountability.md#flip-flopping) attacks. The schematic below explains one scenario where an amnesia attack, a form of flip flopping, can occur such that C1 and C2 commit different blocks.
+
+![](../imgs/tm-amnesia-attack.png)
```

Comment:
I am excited by this diagram! But I don't totally understand it. Can you explain what (for example) the numbers in circles are? And what the labels on the y-axis are? 

If you wanted to really knock it out of the park, you could write out in text the same steps that this diagram is explaining.

Reply:
They are the order of events. I will edit it so that it is clearer.